src/devtools/client/inspector/computed/panel.ts

```diff
@@ -1,15 +1,17 @@
 const { ComputedViewTool } = require("devtools/client/inspector/computed/computed");
-const InspectorTabPanel = require("devtools/client/inspector/components/InspectorTabPanel");
-
+// const InspectorTabPanel = require("devtools/client/inspector/components/InspectorTabPanel");
+import ComputedApp from "devtools/client/inspector/computed/components/ComputedApp";
 import { createElement, ReactElement } from "react";
+import { Inspector } from "../inspector";
 
 export class ComputedPanel {
   private tool: any;
   provider?: ReactElement;
 
-  constructor(inspector: any) {
+  constructor(inspector: Inspector) {
     this.tool = new ComputedViewTool(inspector, window);
-    this.provider = createElement(InspectorTabPanel, { id: "sidebar-panel-computedview" });
+    // this.provider = createElement(InspectorTabPanel, { id: "sidebar-panel-computedview" });
+    this.provider = createElement(ComputedApp);
```

Comment:
this is super weird. Lets prioritize adding these panels to the react tree and removing these tool classes `ComputedViewTool`.

Reply:
https://github.com/RecordReplay/devtools/issues/1210